Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MASTER.MSG

```diff
@@ -117,14 +117,14 @@
 {300}{}{Duh!}
 
 
-# Master Tell-Me-Abouts generic response
+# Master Tell-Me-Abouts generic response (You're to answer his questions, not vice versa. --BreakinBenny)
 {980}{Mast200}{I am the Master. I don't have to answer to you.}
+{981}{Mast201}{I am not here to answer your questions. You are here to answer mine.}

```

Comment:
Those lines aren't used. The Master defaults to his two generic replies, no matter what.

Reply:
And thus they don't need to be used at all! 😊

Reply:
Because the script needs them.

Reply:
Ouchie. I hope a "reply_str" that's set to -1 skips searching for keywords entirely, maybe then my idea could work.